app/main.py

```diff
@@ -1,2 +1,57 @@
+from typing import Any, Hashable
+from dataclasses import dataclass
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.hash_table: list[None | Node] = [None] * self.capacity
+        self.load_factor = 2 / 3
+        self.threshold = int(self.capacity * self.load_factor)
```

Comment:
The load factor is set to `2 / 3`, which is fine, but ensure that the threshold calculation `int(self.capacity * self.load_factor)` correctly reflects the intended behavior. If the load factor is meant to be a fraction, ensure that the threshold calculation rounds down as expected.